### DISCUSSION TITLE
tabs for different events, labeling each ray() output

### DISCUSSION THREAD

**Discussion by sarmenb** [spatie/laravel-ray#182] [Ideas]
I have 2 ideas let me know what you think. 

1) tabs, ability to have different events or actions in each tab. for example, with laravel one tab for events, one tab for emails, and another tab for exceptions.

2) being able to label each ray() output that is displayed that way i know which output i am seeing.

**Comment by freekmurze** [comment]
3. is already covered, it's that toggle in the top right of the screen

**Reply by sarmenb** [reply]
yea just saw it thanks :)

**Reply by sarmenb** [reply]
how about the other 2, good idea?

**Reply by freekmurze** [reply]
1. tabs: probably not soon as this would make the UI much more complicated, and there are not many others asking for this
2. this is already covered, see our docs

**Reply by sarmenb** [reply]
k thanks
